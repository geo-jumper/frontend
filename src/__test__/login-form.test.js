import React from 'react';
import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

import LoginForm from '../components/login-form';

describe('LoginForm', () => {
  test('Testing that the initial login form state is an empty object', () => {
    let mountedLoginForm = Enzyme.mount(<LoginForm />);

    expect(mountedLoginForm.state()).toEqual({});
  });
  test('Testing that the component renders a SignUp form to the page', () => {
    const wrapper = shallow(
      <form></form>
    );
    expect(wrapper).toMatchSnapshot();
  });
});