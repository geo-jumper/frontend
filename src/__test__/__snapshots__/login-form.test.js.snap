// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm Testing that the component renders a SignUp form to the page 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <form />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": "form",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": "form",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
